src/main/resources/assets/html/partials/tdp-invest-login.html

```diff
@@ -0,0 +1,23 @@
+ <div class="col-md-6 col-md-offset-3">
+     <h2>Login</h2>
+    <div ng-show="error" class="alert alert-danger">{{error}}</div>
```

Comment:
According to html & css conventions, the class attribute should always be specified first on a html element. Can you change in all templates?